components/input/input.css

```diff
@@ -0,0 +1,35 @@
+.basic_inp {
+  border: 4px solid transparent;
+  outline: none;
+  padding: 1rem 2rem;
+  border-radius: 4px;
+  font-size: 1.6rem;
+}
+.basic_inp:focus {
+  border: 4px solid var(--primary-li);
+  outline: none;
+}
+
+.inp_lg {
+  height: 3rem;
+  width: 100%;
+}
+.inp_md {
+  height: 2rem;
+  width: 100%;
+}
+.inp_sm {
+  height: 1rem;
+  width: 100%;
+}
```

Comment:
All your inputs are of width:100%.
So you can add width: 100% to you basic_inp class instead of adding everywhere.